apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -691,51 +692,62 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
             </Grid>
             <Divider hidden/>
             { (selectedProtocol === "oidc" && selectedCertInputType === "jwks") && (
-                <Field.Input
-                    ariaLabel="JWKS endpoint URL"
-                    inputType="url"
-                    name="jwks_endpoint"
-                    label="JWKS endpoint URL"
-                    required={ false }
-                    maxLength={ JWKS_URL_LENGTH.max }
-                    minLength={ JWKS_URL_LENGTH.min }
-                    width={ 15 }
-                    initialValue={ EMPTY_STRING }
-                    placeholder="Enter JWKS endpoint URL"
-                    data-testid={ `${ testId }-form-wizard-oidc-jwks-endpoint-url` }
-                />
+                <>
+                    <Field.Input
+                        ariaLabel="JWKS endpoint URL"
+                        inputType="url"
+                        name="jwks_endpoint"
+                        label="JWKS endpoint URL"
```

Comment:
Let's add i18n placeholders.
```suggestion
                        label="JWKS endpoint URL"
```

Reply:
This will be addressed in an upcoming pull request. Hence, resolving this.